src/App.js

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar/Navbar';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
+
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
```

Comment:
- [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page. You would need to add the prop [exact](https://v5.reactrouter.com/web/api/Route/exact-bool) to his route.

Reply:
Thank you so much for the information i didn't knew it before, I'll implement it right now 😁